lib/http_server_fixture/handlers/options.ex

```diff
@@ -0,0 +1,13 @@
+defmodule HTTPServerFixture.Options do
+  alias HTTPServer.Request
+  @behaviour HTTPServer.Handler
+
+  @impl HTTPServer.Handler
+  def handle(%Request{method: "GET"} = _req) do
+    {200, ""}
+  end
+
+  def handle(%Request{method: _method} = _req) do
+    {:error}
+  end
```

Comment:
Again, there is a pattern here where the same lines of code are being added to every single implementation of a Handler.  Also, these same lines of code would need to be required in all future Handlers.  This is a smell known as fan-out.  When it happens, you should look more into the design and see if there is a better pattern to use, or if this responsibility can be moved elsewhere and implemented only once.